UMS.launch

```diff
@@ -11,7 +11,7 @@
     </listAttribute>
     <stringAttribute key="org.eclipse.jdt.launching.CLASSPATH_PROVIDER" value="org.eclipse.m2e.launchconfig.classpathProvider"/>
     <stringAttribute key="org.eclipse.jdt.launching.MAIN_TYPE" value="net.pms.PMS"/>
-    <stringAttribute key="org.eclipse.jdt.launching.PROJECT_ATTR" value="UniversalMediaServer"/>
+    <stringAttribute key="org.eclipse.jdt.launching.PROJECT_ATTR" value="ums"/>
```

Comment:
```suggestion
    <stringAttribute key="org.eclipse.jdt.launching.PROJECT_ATTR" value="UniversalMediaServer"/>
```

Reply:
@SubJunk We had a PR #3535 for naming the project attribute "ums". Do we have a new opinion on that?